api_tests/dry/lightning_routes.ts

```diff
@@ -0,0 +1,67 @@
+// These are stateless tests -- they don't require any state of the cnd and they don't change it
+// They are mostly about checking invalid request responses
+import "chai/register-should";
+import "../lib/setup_chai";
+import { oneActorTest } from "../lib/actor_test";
+import { expect } from "chai";
+
+setTimeout(async function() {
+    describe("Lightning routes", () => {
+        oneActorTest(
+            "lightning-routes-post-eth-lnbtc-return-400",
+            async function({ alice }) {
+                return expect(
+                    alice.cnd.postHanEthereumEtherHalightLightningBitcoin()
```

Comment:
Nit: Can we have that on a separate line please? :see_no_evil: 

So that it is separated from the assertion..

Reply:
Yes. It had that initially but it took me many attempts to hack chai